rear-panel/firmware/main.cpp

```diff
@@ -83,19 +83,21 @@ class LED_BLINK_TASK {
 };
 
 static auto lbt = LED_BLINK_TASK{"Blinkenlights"};
+static auto light_hardware = light_control_hardware::LightControlHardware();
 
 auto main() -> int {
     HardwareInit();
     // RCC_Peripheral_Clock_Select();
     utility_gpio_init();
+    light_hardware.initialize();
 
     // initialize_leds();
```

Comment:
We don't need this anymore

Reply:
Talked in person, deleted the commented out `initialize_leds` call